src/libcrun/handlers/krun.c

```diff
@@ -154,19 +217,38 @@ libkrun_configure_container (void *cookie arg_unused, enum handler_configure_pha
   if (UNLIKELY (ret < 0))
     return ret;
 
+  if (create_sev)
+    {
+      ret = libcrun_create_dev (container, devfd, &sev_device, is_user_ns, true, err);
+      if (UNLIKELY (ret < 0))
+        return ret;
+    }
+
   return 0;
 }
 
 static int
 libkrun_load (void **cookie, libcrun_error_t *err arg_unused)
 {
+  struct krun_config *kconf;
   void *handle;
 
-  handle = dlopen ("libkrun.so.1", RTLD_NOW);
-  if (handle == NULL)
-    return crun_make_error (err, 0, "could not load `libkrun.so.1`: %s", dlerror ());
+  kconf = malloc(sizeof(struct krun_config));
```

Comment:
Is there a cost to loading them both?  As opposed to only loading the one you want?

Reply:
This is certainly going to consume some cycles, and some memory for each found library.